crates/air-lib/test-utils/src/executed_state.rs

```diff
@@ -137,7 +137,12 @@ pub fn ap(generation: u32) -> ExecutedState {
     ExecutedState::Ap(ap_result)
 }
 
-pub fn canon(canonicalized_element: Vec<u8>) -> ExecutedState {
+pub fn canon(_canonicalized_element: Vec<u8>) -> ExecutedState {
```

Comment:
Variable is unused. Is it valid? Is the function used?

Reply:
Thanks, forgot to delete.